Cannot bundle with webpack

> npm i  @ethersphere/bee-js webpack webpack-cli

> src/index.js
```js
import { Bee } from '@ethersphere/bee-js'
console.log(Bee)
```

And run `npx webpack`, it reports:

```
WARNING in ./src/index.js 2:12-15
export 'Bee' (imported as 'Bee') was not found in '@ethersphere/bee-js' (module has no exports)

ERROR in ./node_modules/.pnpm/@ethersphere+bee-js@3.3.0/node_modules/@ethersphere/bee-js/dist/mjs/index.js 1:0-28
Module not found: Error: Can't resolve './bee' in 'C:\Users\Jack\Workspace\bee\node_modules\.pnpm\@ethersphere+bee-js@3.3.0\node_modules\@ethersphere\bee-js\dist\mjs'
Did you mean 'bee.js'?
BREAKING CHANGE: The request './bee' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ./src/index.js 1:0-41 2:12-15
```

It looks like this package ships an invalid ESM bundle. (Node ESM requires a fully specified extension name).

--- a/.depcheckrc.json
+++ b/.depcheckrc.json
@@ -2,6 +2,7 @@
   "ignores": [
     "@babel/core",
     "@babel/plugin-proposal-class-properties",
+    "babel-plugin-add-import-extension",
     "@babel/plugin-transform-runtime",
     "@babel/preset-env",
     "@babel/preset-typescript",

--- a/package.json
+++ b/package.json
@@ -47,7 +47,7 @@
   "scripts": {
     "prepublishOnly": "cross-env NODE_ENV=production npm run build",
     "build": "rimraf dist && npm run build:node && npm run build:types && npm run build:browser",
-    "build:node": "tsc -p tsconfig.json && tsc -p tsconfig-mjs.json && ./build-fixup",
+    "build:node": "tsc -p tsconfig.json && tsc -p tsconfig-mjs.json && ./build-fixup && babel --plugins \"babel-plugin-add-import-extension\" --out-dir dist/mjs/ dist/mjs/",
     "build:types": "tsc --emitDeclarationOnly --declaration --outDir dist/types",
     "build:browser": "webpack --progress",
     "mock-ci": "npm run lint:check && npm run check:types && npm run test",
@@ -88,6 +88,7 @@
     }
   },
   "devDependencies": {
+    "@babel/cli": "^7.17.0",
     "@babel/core": "^7.14.6",
     "@babel/plugin-proposal-class-properties": "^7.16.7",
     "@babel/plugin-transform-runtime": "^7.14.5",
@@ -113,6 +114,7 @@
     "@typescript-eslint/parser": "^5.10.0",
     "babel-jest": "^27.0.5",
     "babel-loader": "^8.2.2",
+    "babel-plugin-add-import-extension": "^1.6.0",
     "debug": "^4.3.1",
     "depcheck": "^1.4.3",
     "eslint": "^8.8.0",

--- a/src/modules/debug/status.ts
+++ b/src/modules/debug/status.ts
@@ -2,7 +2,7 @@ import { http } from '../../utils/http'
 import type { Health, NodeInfo } from '../../types/debug'
 import { Ky } from '../../types'
 import { BeeVersions } from '../../types/debug'
-import getMajorSemver from 'semver/functions/major'
+import getMajorSemver from 'semver/functions/major.js'
 
 // Following lines bellow are automatically updated with GitHub Action when Bee version is updated
 // so if you are changing anything about them change the `update_bee` action accordingly!

--- a/src/types/web-streams-polyfill/index.d.ts
+++ b/src/types/web-streams-polyfill/index.d.ts
@@ -1,0 +1,6 @@
+/**
+ * This is type "alias" for the direct import of the web-streams-polyfill/dist/ponyfill.js
+ */
+declare module 'web-streams-polyfill/dist/ponyfill.js' {
+  export * from 'web-streams-polyfill/ponyfill'
+}

--- a/src/utils/stream.ts
+++ b/src/utils/stream.ts
@@ -1,6 +1,6 @@
 import { Readable as NodeReadableNative, ReadableOptions as NodeReadableOptions } from 'stream'
 import { isStrictlyObject } from './type'
-import { ReadableStream } from 'web-streams-polyfill/ponyfill'
+import { ReadableStream } from 'web-streams-polyfill/dist/ponyfill.js'
 import { Readable as NodeReadableStream } from 'readable-stream'
 
 import { Readable } from '../types'

